aea/test_tools/test_cases.py

```diff
@@ -170,6 +174,20 @@ def _run_python_subprocess(cls, *args: str, cwd: str = ".") -> subprocess.Popen:
         cls.subprocesses.append(process)
         return process
 
+    @classmethod
+    def start_subprocess(cls, *args: str, cwd: str = ".") -> subprocess.Popen:
```

Comment:
I wonder, why not make it more general? The `args` could just be forwarded to `Popen`.
(Maybe it already exists, but not sure, worth checking).

Then, you can define a new function `call_python_interpreter`, or something like that, that prepend the `sys.executable` and calls the generic `start_subprocess(sys.executable, *args)`

Reply:
Fair, but tbh I don't want to work any more on this class atm. We can do it when we need it.